stats/view/worker.go

```diff
@@ -109,12 +110,12 @@ func Unsubscribe(views ...*View) {
 	<-req.done
 }
 
-// Deprecated: Use the Unsubscribe function instead.
-func (v *View) Unsubscribe() error {
+// Deprecated: Use the Unregister function instead.
+func (v *View) Unregister() error {
```

Comment:
This should not be renamed since the method only exists for backwards compatibility

Reply:
Roger that.